build.gradle

```diff
@@ -70,15 +70,14 @@ apply plugin: 'opensearch.testclusters'
 apply plugin: 'opensearch.pluginzip'
 
 def sqlJarDirectory = "$buildDir/dependencies/opensearch-sql-plugin"
-def jsJarDirectory = "$buildDir/dependencies/opensearch-job-scheduler"
-def adJarDirectory = "$buildDir/dependencies/opensearch-anomaly-detection"
+def adJarDirectory = "$buildDir/dependencies/opensearch-time-series-analytics"
```

Comment:
Any reference to time-series-analytics to be reverted to anomaly-detection. The name change is only tracked on `main`.

Reply:
done